c8ylp/cli/core.py

```diff
@@ -35,7 +35,7 @@
 from ..banner import BANNER1
 from ..env import save_env
 from ..rest_client.c8yclient import CumulocityClient, CumulocityMissingTFAToken
-from ..tcp_socket import TCPProxyServer
+from ..tcp_socket import TCPProxyServer, UnixStreamProxyServer
```

Comment:
This import makes the Windows Unit Test fail:
c8ylp\cli\core.py:38: in <module>
    from ..tcp_socket import TCPProxyServer, UnixStreamProxyServer
c8ylp\tcp_socket\__init__.py:20: in <module>
    from c8ylp.tcp_socket.tcp_server import TCPProxyServer, UnixStreamProxyServer
c8ylp\tcp_socket\tcp_server.py:123: in <module>
    class CustomUnixStreamServer(socketserver.UnixStreamServer):
E   AttributeError: module 'socketserver' has no attribute 'UnixStreamServer'

We can check the socket or the OS and make a conditional import
```suggestion
import platform
if 'Linux' == platform.system():
    from ..tcp_socket import UnixStreamProxyServer
from ..tcp_socket import TCPProxyServer
```

Reply:
Refactored, the unixstreamproxyserver lives in its own module now, and the imports are check the AF_UNIX attribute of the socket module, this is what they also do in the socketserver module.